api/features/migrations/0060_set_featurestate_environment_not_null.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.2.20 on 2023-08-30 13:14
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def delete_feature_states_without_environment(apps, schema_editor):
+    FeatureState = apps.get_model("features", "FeatureState")
+    FeatureState.objects.filter(environment=None).delete()
```

Comment:
I don't know if it makes a difference, but the correct way to do this I think is: `filter(environment__isnull=True)`. I think it's perhaps because in different database engines, it handles `null` differently.

Reply:
Yeah, done